api/Models/RoomToAdd.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace iTechArt.Hotels.Api.Models
+{
+    public class RoomToAdd
+    {
+        [MaxLength(150)]
+        public string Name { get; set; }
+        public int Sleeps { get; set; }
+        public int Number { get; set; }
+        [Column(TypeName = "decimal(19,4)")]
```

Comment:
What is the purpose for this attribute in model that is used as an action parameter?
Also shouldn't we have some additional validation for all the fields?